.github/workflows/test_galaxy_packages_for_pulsar.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.7']
+        python-version: ['3.8']
```

Comment:
Can you revert this please? We need to keep testing these packages on 3.7 (that's the whole point of this workflow, see details in #16394).